SITCOMTN-023.tex

```diff
@@ -143,7 +187,8 @@ \subsection{Workflows}
 \subsection{Ticket Creation and Review process}
 To enforce a common process, the workflow has constraints on who is allowed to create tickets and transition them to different state transition. 
 
-Only SIT-COM leads can create, deprecate and/or reopen Milestones tickets. SCLT coordinators and leads can create an Epic. 
+Only SIT-COM leads can create, deprecate and/or reopen Milestones tickets. 
+SCLT coordinators and leads can create an Epic. 
 However, only the assignee can invalidate, deprecate, reopen and/or complete the Epic. 
 Stories/bugs are created by a SIT-Com member and the tasks by anyone on the project. 
 
```

Comment:
I would add "any" in from of the SIT-Com leads and SCLT coordinators. 
I would also change the permissions of an epic as well. Any SCLT leads and coordinators along with the assingee should have the right tinvalidate, deprecate, reopen and/or complete Epics.

Reply:
So the change I made here was just to put the sentences on different lines (semantic line versioning). I didn't change any content.
Can we make this part of a different ticket? It will mean changing the table as well and taking a deeper look at the workflow.